lib/mitie/trainers/ner.rb

```diff
@@ -0,0 +1,55 @@
+module Mitie
+  module Trainers
+    class NER
+      def initialize(filename)
+        raise ArgumentError, "File `#{filename}' does not exist" unless File.exist?(filename)
+        @pointer = FFI.mitie_create_ner_trainer(filename)
+
+        ObjectSpace.define_finalizer(self, self.class.finalize(@pointer))
+      end
+
+      def add(instance)
+        FFI.mitie_add_ner_training_instance(@pointer, instance.pointer)
+      end
+
+      def beta
+        FFI.mitie_ner_trainer_get_beta(@pointer)
+      end
+
+      def beta=(value)
+        raise ArgumentError, "beta must be greater than or equal to zero" unless value >= 0
+
+        FFI.mitie_ner_trainer_set_beta(@pointer, value)
+        nil
```

Comment:
Ruby always returns `value` from methods that end in `=`, so no need for `nil` here and below.